test/create_vm_from_template_test.go

```diff
@@ -310,7 +310,7 @@ var _ = Describe("Create VM from template", func() {
 	})
 
 	Context("with StartVM", func() {
-		DescribeTable("VM is created from template with StartVM attribute", func(config *testconfigs.CreateVMTestConfig, phase kubevirtv1.VirtualMachineInstancePhase, running bool) {
+		DescribeTable("VM is created from template with StartVM attribute", func(config *testconfigs.CreateVMTestConfig, phase kubevirtv1.VirtualMachineInstancePhase, runStrategy kubevirtv1.VirtualMachineRunStrategy) {
```

Comment:
The Entries below are still passing in bools instead of run strategies.

Reply:
Yep, forgot to update it. Thanks Felix!